integration_tests/tests/web_app_certos.rs

```diff
@@ -465,6 +465,35 @@ mod website {
       check_public_certificate(&d, &title, &format!("Invitation {raw_description}"), &image).await;
     }
 
+
+    integration_test!{ use_graphiql (c, d)
+      async fn send_graphql_query(d : &Selenium, query: &str) {
+        d.goto("http://localhost:8000/#").await;
+        d.click("#graphiql").await;
+        d.driver.maximize_window().await.expect("to maximize window");
```

Comment:
This "maximize_window" should be added to all tests then.

Reply:
the following line is added in the function "start" implemented for the struct Selenium inside the selenium.rs file: 

`driver.maximize_window().await.expect("to maximize window");`